solidity-v1/dashboard/src/pages/threshold/HowItWorksPage.jsx

```diff
@@ -0,0 +1,136 @@
+import React from "react"
+import NavLink from "../../components/NavLink"
+import * as Icons from "../../components/Icons"
+import List from "../../components/List"
+import { LINK } from "../../constants/constants"
+
+const HowItWorksPage = () => {
+  return (
+    <section className="threshold-how-it-works-page">
+      <section className="tile threshold-how-it-works-page__explanation">
+        <Icons.KeepTUpgrade className="threshold-how-it-works-page__explanation__icon" />
```

Comment:
Let's update this icon. It has been changed- please see https://www.figma.com/file/Su61MwHLrQn3ld24q85pMx/Token-Dashboard-dApp?node-id=12483%3A97919

Reply:
Fixed in c9c0a032de3dd34a6ae32044b5b3e8970a355d5d